docker-compose.yaml

```diff
@@ -1,6 +1,21 @@
 version: "3.7"
 
 services:
+  single-validator:
+    build:
+      context: chain
+      dockerfile: Dockerfile
+    image: band-validator:latest
+    environment:
+      - PRIVATE_KEY=06be35b56b048c5a6810a47e2ef612eaed735ccb0d7ea4fc409f23f1d1a16e0b
+      - NODE_URI=tcp://localhost:26657
+    networks:
+      bandchain:
+        ipv4_address: 172.18.0.11
+    ports:
+      - 26657:26657
+    command: sh -c "chmod +x ./docker-config/single-validator/run.sh && ./docker-config/single-validator/run.sh"
+
```

Comment:
We can define `multi-validator` as a dummy service. See https://github.com/docker/compose/issues/2589#issuecomment-313796200 for example.